sirepo/template/template_common.py

```diff
@@ -230,7 +230,7 @@ def validate_model(model_data, model_schema, enum_info):
                 'BeamList', 'MirrorFile', 'ImageFile', 'String', 'OptionalString', 'MagneticZipFile',
```

Comment:
Should the list include the 'Float' and 'Integer' types, even though they get handled separately?